scripts/server.sh

```diff
@@ -4,14 +4,6 @@ set -e
 
 echo "Run the RWD Docker container."
 
-if [ -z "$RWD_DATA" ]; then
```

Comment:
Right, but if the `$RWD_DATA` environment variable isn't set nothing will go to the `-e` flag:
```
+ docker-compose run --rm -e RWD_DATA= --entrypoint /bin/bash app
WARNING: The RWD_DATA variable is not set. Defaulting to a blank string.
ERROR: Cannot create container for service app: create .: "." includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed
```